tests/contrib/gunicorn/test_gunicorn.py

```diff
@@ -170,11 +170,12 @@ def gunicorn_server(gunicorn_server_settings, tmp_path):
     debug_mode=True,
     enable_module_cloning=True,
 )
-SETTINGS_GEVENT_SPANAGGREGATOR_RLOCK = _gunicorn_settings_factory(
+# default is RLOCK, this tests to make sure Lock still works
+SETTINGS_GEVENT_SPANAGGREGATOR_LOCK = _gunicorn_settings_factory(
```

Comment:
nit: We can be more explicit here. If the name of this test case is clear we may not need a the comment above. 

```suggestion
SETTINGS_GEVENT_SPANAGGREGATOR_NO_RLOCK = _gunicorn_settings_factory(
```